cdk/infra.ts

```diff
@@ -168,5 +169,27 @@ systemctl start ${app}
       stringValue: ec2.listener.listenerArn,
     });
 
+    // Create IAM role to allow Galaxies data-refresher-lambda to write to relevant portion of the bucket
+    // https://github.com/guardian/galaxies
+    const devPlaygroundAccountId = StringParameter.fromStringParameterAttributes(this, "devPlaygroundAccountId", {
+      parameterName: `${configPrefix}/developerPlaygroundAccountId`,
+    }).stringValue;
```

Comment:
Recommend using [private-infrastructure-config](https://github.com/guardian/private-infrastructure-config/blob/de75e57d7e9e883b02108eb48692af87080171ba/src/aws-accounts.ts#L20) here, rather than creating an additional SSM Parameter.

Reply:
Done, thank you @akash1810!